rpcs3/Emu/System.h

```diff
@@ -458,6 +472,22 @@ struct cfg_root : cfg::node
 
 	} net{this};
 
+	struct node_gdb : cfg::node
+	{
+		node_gdb(cfg::node* _this) : cfg::node(_this, "GDB debug server") {}
+
+		cfg::_bool enabled{this, "Enabled", true};
+		cfg::string socket_type{this, "Socket type", "INET"};
+		cfg::_int<1, 65535> ipv4_port{this, "IPv4 port", 2345};
+		cfg::string unix_fpath{this, "UNIX socket file path",
+#ifndef _WIN32
```

Comment:
Makes more sense to hide the whole config string

Reply:
Clarify what you mean?

Reply:
he means remove the option completely on windows